packages/ui/core-components/src/lib/unsorted/viz/core/Value.svelte

```diff
@@ -25,6 +27,10 @@
 
 	// Remove any undefined props (e.g. w/o defaults) to prevent them from being passed
 	$: spreadProps = Object.fromEntries(Object.entries($$props).filter(([, v]) => v !== undefined));
+
+	$: if (agg) {
+		data = data.groupBy(undefined).agg({ [agg]: { col: 'fare', as: [column] } });
+	}
```

Comment:
What happens if I want to get the total (or avg) distance travelled by plane, rather than fare?